### PR TITLE
Logging Overhaul

### PR DESCRIPTION
**Pull request by RushilShaw**
Changed Logging to use Winston
Reduced Verbosity of Logging
Changed Logging to work with inputs including "0", "1", and "2" to represent "error", "info", "debug" respectively
Made Minor changes to logging messages

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,8 @@
     "axios": "^1.7.7",
     "dotenv": "^16.4.5",
     "form-data": "^4.0.0",
-    "ts-node": "^10.9.2"
+    "sqlite3": "^5.1.7",
```
Comment (galaxybomb23) [inline package.json:40]: why are we adding sqlite3?

**Review by galaxybomb23 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (RushilShaw) [reply package.json:40]: I'm not sure. I didn't npm install it. My only thinking is that it is a dependency of winston.

Comment (galaxybomb23) [reply package.json]: i think you once again pulled from OLD main instead of most UpToDate branch (#1 )

**Comment by RushilShaw** [comment]
After the 6th commit, the output for ./run test is

39/46 test cases passed. 75% line coverage achieved.

**Review by Coop8 — APPROVED** [review]
Goodbye proprietary logger, welcome back Winston!

**Pull request merged**